### PR TITLE
Qtel: Add AppStream metadata, new icon

### PR DESCRIPTION
**Pull request by AsciiWolf**
This Pull Request adds [AppStream](https://www.freedesktop.org/software/appstream/docs/) metadata (to allow Qtel to be found and installed using GNOME Software, KDE Discover and other modern package management tools that depend on AppStream) and also adds a new, much better quality/resolution Qtel desktop icon that is required by AppStream.

The desktop icon is taken from this Debian package: https://packages.debian.org/sid/qtel-icons
According to the "copyright" file of that package, the icon license is GPL-2+.

This PR also changes the Icon in a desktop file to a generic "qtel" one instead of a hardcoded file path and also does the same thing for Exec binary.

Feel free to review and let me know if there is any problem or if you have any question. Thanks!

### CODE REVIEW

File: src/qtel/CMakeLists.txt [hunk 1]
```diff
@@ -70,6 +70,8 @@ install(TARGETS ${PROG} DESTINATION ${BIN_INSTALL_DIR})
 install(FILES connect.raw DESTINATION ${SHARE_INSTALL_PREFIX}/qtel/sounds)
 install(FILES qtel.desktop DESTINATION ${SHARE_INSTALL_PREFIX}/applications)
 install(FILES images/link.xpm DESTINATION ${SHARE_INSTALL_PREFIX}/icons)
```
Comment (AsciiWolf) [inline src/qtel/CMakeLists.txt:72]: By the way, this is probably not the best location for xpm icons. The correct one should be `${SHARE_INSTALL_PREFIX}/pixmaps`. But I am not sure if this location is not hardcoded in the Qtel source code, therefore I have decided not to change it. Feel free to fix it if you want or let me know and I can fix it as a part of this PR.

Comment (AsciiWolf) [reply src/qtel/CMakeLists.txt]: Done (as a second commit). :-)

**Comment by AsciiWolf** [comment]
@sm0svx Feel free to let me know if there is anything else that I should change or fix. Thanks! :-)

**Comment by sm0svx** [comment]
So have you tested that Qtel will find all icons after your changes?  No code changes needed?

**Comment by AsciiWolf** [comment]
@sm0svx I have just tested it on my computer and all icons seem to be displayed correctly.

It actually looks like that the _/usr/share/icons/link.xpm_ (_/usr/share/pixmaps/link.xpm_) icon is not used anywhere in the code and since this PR replaces it with the png icon, I am almost certain that it can be safely removed from the CMake install script (and Fedora spec file). I will push a new commit to do this.

**Comment by AsciiWolf** [comment]
@sm0svx Done, everything should be ready to be merged. :-) Feel free to do a final review.

**Pull request merged**